torch_np/tests/numpy_tests/core/test_multiarray.py

```diff
@@ -4549,6 +4550,7 @@ def test_none_shape(self):
         x.resize()
         assert_array_equal(x, np.eye(3))
 
+    @pytest.mark.xfail(reason="why would one do it")
```

Comment:
Turns out `ndarray.resize` is documented to differ from `np.resize`, so many of these corner cases are in fact not corner cases but documented differences:

> If the new array is larger than the original array, then the new array is filled with repeated copies of a. Note that this behavior is different from a.resize(new_shape) which fills with zeros instead of repeated copies of a.

https://numpy.org/doc/stable/reference/generated/numpy.resize.html

One other annoyance is the support of both `resize(tuple)` and `resize(*tuple)`. Anyhow, this is all done in gh-106.